### ISSUE TITLE
5.3.0 does not create a schemas/dask-labextension/plugin.json file on install

### ISSUE THREAD

**Issue by theNewFlesh** [dask/dask-labextension#245]
**Dask lab extension does not show up in Jupyter Lab settings**
I have messed with a bunch of config settings and still cannot get this to work.

**Environment**:
docker container running:
 - ubuntu                               18.04
 - python                                  3.7
 - dask                            2022.2.0
 - dask-gateway                   0.9.0
 - dask-labextension             5.3.0
  - jupyterhub                         3.0.0
  - jupyterlab                      3.5.0rc0

and a docker container running:
 - ubuntu                             22.04
 - python                               3.10
  - dask                          2022.6.1
  - dask-gateway                  2022.10.0
  - dask-labextension             5.3.0
  - jupyterhub                    3.0.0
  - jupyterlab                    3.5.0rc0

- Install method: pip3

**Comment by theNewFlesh** [comment]
```
>>> jupyter lab --ip=0.0.0.0 --no-browser
[W 2022-10-20 21:15:15.497 LabApp] wrote error: 'Schema not found: /home/ubuntu/.local/share/jupyter/lab/schemas/dask-labextension/plugin.json'
```

**Comment by theNewFlesh** [comment]
Appears that the schema is not valid or something?

**Comment by theNewFlesh** [comment]
Copying plugin.json from `pangeo/base-notebook:latest` in `/srv/conda/envs/notebook/share/jupyter/labextensions/dask-labextension/schemas/dask-labextension` to `~/.local/share/jupyter/labextensions/dask-labextension/schemas/dask-labextension` fixes the issue.

Not sure why `pip3 install dask-labextension` doesn't do that in the first place.

**Comment by theNewFlesh** [comment]
`pip3.10 install dask-labextension==5.2.0` creates a plugin.json file in the schema directory.
`pip3.10 install dask-labextension==5.3.0` does not create a correct plugin.json file in the schema directory.

**Comment by ian-r-rose** [comment]
Thank you for the report @theNewFlesh, I can reproduce your issue. I'm not sure just yet of the cause. Interestingly, the problem doesn't seem to appear for the `conda-forge` packages, just the ones on pypi. But when I inspect the tarballs, the `plugin.json` file appears to be there.

**Comment by theNewFlesh** [comment]
yup

**Comment by theNewFlesh** [comment]
my dockerfile hotfix
```
RUN echo "\n${CYAN}REPAIR DASK-LABEXTENSION${CLEAR}"; \
    mkdir -p /tmp/dask-labextension && \
    cd /tmp/dask-labextension && \
    wget \
        https://github.com/dask/dask-labextension/archive/refs/tags/5.3.0.tar.gz \
        -O dask-labextension.tar.gz && \
    tar xvf dask-labextension.tar.gz && \
    cp \
        `find . -type f | grep schema/plugin.json` \
        /home/ubuntu/.local/share/jupyter/labextensions/dask-labextension/schemas/dask-labextension/ && \
    rm -rf /tmp/dask-labextension
```

**Comment by theNewFlesh** [comment]
Node version doesn't seem to affect the issue since I reproduced it with the default version 12 and custom installed version 16 on ubuntu 22.04. A thought..perhaps the schema dir is referenced as "schema" in some places and "schemas" in others?

**Comment by stonebig** [comment]
It happens also on windows pip install, apparently, with dask_labextension-5.3.0-py3-none-any.whl.... a late discovery

**Comment by ian-r-rose** [comment]
I plan to make another release today to fix this

**Comment by ian-r-rose** [comment]
I just published a patch release 5.3.1 to PyPI which should fix this. Sorry about the bad wheel, and let me know if you hit any other problems here!
